### ISSUE TITLE
specify: event

### ISSUE THREAD

**Issue by MortenHofft** [gbif/model-material#52]
ERROR:  null value in column "event_type" of relation "event" violates not-null constraint
DETAIL:  Failing row contains (7f034432-c814-4d6d-9ed7-64ff5dff6338, null, https://www.gbif.org/dataset/8f79c802-a58c-447f-99aa-1d6a0790825..., a781c880-72ea-4c35-8539-cf95d971a4ef, null, null, null, DE-N-236, 3/2/1996, 1996, 3, 2, null, null, null, null, null, 28.3999996° N, 81.1999969° E, null, null, null, null, null, null, null, null, null).
CONTEXT:  COPY event, line 2: "7f034432-c814-4d6d-9ed7-64ff5dff6338,,https://www.gbif.org/dataset/8f79c802-a58c-447f-99aa-1d6a07908..."
SQL state: 23502

**Comment by MortenHofft** [comment]
https://github.com/gbif/model-material/blob/master/schema.sql#L141
event_type must be filled

**Comment by acbentley** [comment]
What should this contain?  I do not see any guidance anywhere and not a DwC field.

**Comment by MortenHofft** [comment]
I'm just trying to import the data, so my focus is simply on that it is valid and not on the content. 

That being said, I think you can just invent something. I've seen `OCCURRENCE` and `GATHERING` being used so far. But I'm not sure how much could be an event. A blood sample? A measuring event? @tucotuco is better at answering questions like that than me

**Comment by tucotuco** [comment]
A reasonable target would be to use the controlled [vocabulary for eventType](https://registry.gbif-uat.org/vocabulary/EventType/concepts) found in the GBIF vocabulary server. That vocabulary is a work in progress, but it might not be a bad idea to get in the habit of following those vocabs. So, in your case, I think the closest match from the vocabulary would be "Collect".

**Comment by tucotuco** [comment]
I just realized we gave specific guidance in the [Occurrence and Other Events section](https://github.com/gbif/model-material/blob/master/data-mapping.md#12-occurrences-and-other-events) of the Mapping for Events that are Occurrences:

Each Occurrence has its own occurrenceID. The Occurrences associated with a given Organism can be discovered by the organismID they have in common. Every Occurrence must have a corresponding Event record in which the eventID is the same is the occurrenceID and the eventType for the Event record MUST be OCCURRENCE.
